### ISSUE TITLE
SCSS compilation fails after upgrading to Angular 10

### ISSUE THREAD

**Issue by chase9** [telerik/kendo-themes#1938]
**Describe the bug**

Upon upgrading my application to Angular 10, I started receiving errors resolving the truetype font when compiling SCSS. Here's the error:

```
ERROR in ./src/kendo-dark.scss
Module build failed (from ./node_modules/mini-css-extract-plugin/dist/loader.js):
ModuleError: Module Error (from ./node_modules/postcss-loader/src/index.js):
(Emitted value instead of an instance of Error) CssSyntaxError: <src path>/node_modules/@progress/kendo-theme-default/scss/icons/_layout.scss:10:8: Can't resolve ') format("truetype"' in '<src path>/mdDashboard/src'

   8 |         font-style: normal;
   9 |         font-weight: normal;
> 10 |         src: url(#{$icon-font-url}) format( "truetype" );
     |        ^
  11 |     }
  12 | 

    at Object.emitError (<src path>/mdDashboard/node_modules/webpack/lib/NormalModule.js:173:6)
    at <src path>/mdDashboard/node_modules/@angular-devkit/build-angular/src/angular-cli-files/plugins/postcss-cli-resources.js:143:28
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
```

**To reproduce**

1. Generate an Angular project set to use scss
2. Create a custom theme for dark mode (I've supplied my Kendo styles)
3. Attempt to run `ng start` and wait for it to fail with above error.

**Expected behavior**

The SCSS should continue to compile successfully.

**Screenshots**

**Affected package (please remove the unneeded items)**
* theme-default

**Affected suites (please remove the unneeded items)**
* Kendo UI for Angular

**Affected browsers (please remove the unneeded items)**
* Not Applicable

**Build system information (please remove the unneeded items)**
* OS: Mac
* Node version: v12.18.2
* NPM version: 6.14.8

**Additional context**
Add any other context about the bug.

The issue is caused by line 10 in @progress/kendo-theme-default/scss/icons/_layout.scss. Changing
`src: url(#{$icon-font-url}) format( "truetype" );`
to
`src: url(#{$icon-font-url}) //format( "truetype" );`
Will allow the program to compile and start.

I am using a custom webpack-config to remove extra languages from moment.js, but this has not caused any problems before.

**Comment by chase9** [comment]
Here's my Kendo styles, renamed to .txt from .scss:

[kendo-dark.txt](https://github.com/telerik/kendo-themes/files/5313047/kendo-dark.txt)
[kendo.txt](https://github.com/telerik/kendo-themes/files/5313048/kendo.txt)

**Comment by chase9** [comment]
Since Angular 11 is now out, I spent some time trying to fix this issue so we can upgrade our project to Angular 10. I ended up fixing it by adding this variable to my kendo-dark.scss:

`$icon-font-url: "~@progress/kendo-theme-default/scss/icons/font/WebComponentsIcons.ttf";`

It appears that, starting with Angular 10, there is an issue resolving the path to the icon fonts when you have a theme imported into another theme. I'm not sure if you'd like me to leave this issue open or closed since I fixed it?
